L117-cpp-replacing-gpr-logging-with-abseil-logging.md

```diff
@@ -0,0 +1,94 @@
+L117-Replacing gpr logging with Abseil logging
+----
+
+* Author(s): tjagtap@google.com
+* Approver: roth@google.com , ctiller@google.com
+* Status: In Review 
+* Implemented in: gRPC C++
+* Last updated: April 29, 2024.
+* Discussion at: (filled after thread exists)
```

Comment:
Please create the mailing list thread and link to it here, as per the gRFC process.